dev/CommonStyles/Common_themeresources_any.xaml

```diff
@@ -659,17 +661,113 @@
             <SolidColorBrush x:Key="CircleElevationBorderBrush" Color="{ThemeResource SystemColorWindowTextColor}" />
             <SolidColorBrush x:Key="AccentControlElevationBorderBrush" Color="{ThemeResource SystemColorWindowTextColor}" />
 
-            <!-- System high contrast colors -->
+            <!-- Colors copied from Default theme to match list of keys -->
+
+            <Color x:Key="TextFillColorPrimary">#FFFFFF</Color>
+            <Color x:Key="TextFillColorSecondary">#C5FFFFFF</Color>
+            <Color x:Key="TextFillColorTertiary">#87FFFFFF</Color>
+            <Color x:Key="TextFillColorDisabled">#5DFFFFFF</Color>
+            <Color x:Key="TextFillColorInverse">#E4000000</Color>
+
+            <Color x:Key="AccentTextFillColorDisabled">#5DFFFFFF</Color>
+            
+            <Color x:Key="TextOnAccentFillColorDefault">#FFFFFF</Color>
+
+            <Color x:Key="TextOnAccentAAFillColorPrimary">#000000</Color>
+            <Color x:Key="TextOnAccentAAFillColorSecondary">#80000000</Color>
+            <Color x:Key="TextOnAccentAAFillColorDisabled">#87FFFFFF</Color>
+
+            <Color x:Key="ControlFillColorDefault">#0FFFFFFF</Color>
+            <Color x:Key="ControlFillColorSecondary">#15FFFFFF</Color>
+            <Color x:Key="ControlFillColorTertiary">#08FFFFFF</Color>
+            <Color x:Key="ControlFillColorDisabled">#0BFFFFFF</Color>
+            <Color x:Key="ControlFillColorTransparent">#00FFFFFF</Color>
+            <Color x:Key="ControlFillColorInputActive">#B31E1E1E</Color>
+
+            <Color x:Key="ControlAAFillColorDefault">#8BFFFFFF</Color>
```

Comment:
A handful of these color names were updated in a PR that was checked in Friday.

https://github.com/microsoft/microsoft-ui-xaml/pull/4628

Reply:
we decided to keep the old ones. I moved those into the deprecated file.